src/config.rs

```diff
@@ -1,3 +1,5 @@
+const PAGE_SIZE: u64 = 4096;
```

Comment:
Looks like there are some unused code warnings about this. It probably needs some cfg attribute.

Reply:
Should be fixed now.